test/e2e-common.sh

```diff
@@ -20,12 +20,34 @@
 E2E_MIN_CLUSTER_NODES=4
 E2E_MAX_CLUSTER_NODES=4
 
+# Change these two knobs to get a different Istio configuration
+ISTIO_VERSION="1.0.6"      # Istio version.
+ISTIO_OPTION="mesh"        # 'mesh' or 'no-mesh'?
+
 # This script provides helper methods to perform cluster actions.
 source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.sh
 
+# Function to choose an Istio version to test with.
```

Comment:
Please also document the parameters.

nit: this comment should be 2 lines below.

Reply:
done